src/proxyp/Elements.h

```diff
@@ -11,6 +11,8 @@
 
 #include "sbuf/SBuf.h"
 
+#include <map>
```

Comment:
It is rather unfortunate that we have to include <map> in what-is-supposed-to-be elementary header file just so that proxyp/Message.cc can iterate pseudo header names. proxyp/Message.cc actually does not care how the pseudo headers are stored. Can you think of a better alternative?

Reply:
No matter what type is used, it needs to be included because it is used in two different .cc. That is the bug being fixed here.

Reply:
Addressing my concern would require changing what type is used in at least one of those .cc files.

Reply:
Untested (beyond building) faf92c2b64e8b62513b548a67c32207c94aa77c8 addresses my concern by hiding PseudoHeaderFields type inside proxyp/Elements.cc. If you like the idea, please retest and re-review. Otherwise, please revert and commit the original fix.